cluster_config/configurations.yaml.tpl

```diff
@@ -37,10 +37,10 @@ Configurations:
     "hive.compactor.initiator.on": "true"
     "hive.compactor.worker.threads": "1"
     "hive.support.concurrency": "true"
-    "javax.jdo.option.ConnectionURL": "jdbc:mysql://${hive_metastore_fqdn}:3306/hive?createDatabaseIfNotExist=true"
+    "javax.jdo.option.ConnectionURL": "jdbc:mysql://${hive_metastore_endpoint}:3306/hive?createDatabaseIfNotExist=true"
```

Comment:
maybe you can do a similar implementation here as seen in adg: https://github.com/dwp/aws-analytical-dataset-generation/blob/7fa2bc80cb9079eda06a01a7b9cf3ff66a9c8fd1/cluster_config/configurations.yaml.tpl#L48